getting_started/devpreview_faq.adoc

```diff
@@ -165,6 +165,12 @@ volume claim (PVC) or an EmptyDir at this time.
 * Only the Source-to-Image (S2I) build strategy is allowed for any build
 configurations imported into your project.
 
+[[devpreview-access-internal-registry]]
+HOW DO I ACCESS THE INTERNAL REGISTRY?::
+See
+link:../dev_guide/managing_images.html#accessing-the-internal-registry[Accessing
+the Internal Registry] for login instructions.
```

Comment:
I liked how you had the URL shown here, just was suggesting also linking to that doc for more details.